doc/manual/io.rst

```diff
@@ -349,6 +351,20 @@ to the fif format with help of the :ref:`mne_eximia2fiff` script.
 It creates a BrainVision ``vhdr`` file and calls :ref:`mne_brain_vision2fiff`.
 
 
+Setting EEG references
+######################
+
+The preferred method for creating an average EEG reference in MNE is
```

Comment:
The preferred method for applying an EEG reference in MNE is func:`mne.set_eeg_reference`, or equivalent instance methods like :meth:`raw.set_eeg_reference() <mne.io.Raw.set_eeg_reference>`.